mlflow/server/js/src/components/ExperimentView.js

```diff
@@ -210,6 +236,9 @@ class ExperimentView extends Component {
     const compareDisabled = Object.keys(this.state.runsSelected).length < 2;
     const deleteDisabled = Object.keys(this.state.runsSelected).length < 1;
     const restoreDisabled = Object.keys(this.state.runsSelected).length < 1;
+    const compactViewButtonClassName = this.state.showMultiColumns ? "" : "active";
```

Comment:
ditto: let's use classnames module here and for the next usage and maybe even inline them.